### DISCUSSION TITLE
Dependencies

### DISCUSSION THREAD

**Discussion by SebGue** [Eomys/pyleecan#655] [Q&A]
Hello @all, 

I've setup a fresh environment and was wondering if there is a doc on why we got version restrictions on some of the dependencies?

Best regards, Sebastian

**Comment by BonneelP** [comment]
Hello,

Sadly this document doesn't exist but that would be a great addition to the project :) 
Hopefully we should soon be able to investigate moving to Python 3.11 and we will take the opportunity to check all the packages versions (including black). The first important package to investigate will be PySide2 => PySide6.

Best regards,
Pierre

**Comment by SebGue** [comment]
Hello @BonneelP ,

is there a reason to use gmsh-sdk, since this package is depreciated? Otherwise we should switch to gmsh or gmsh-dev.

BR, Sebastian

**Comment by BonneelP** [comment]
Hello Sebastian,

Sadly, I don't remember the reason why we started with gmsh-sdk but if it is depreciated we can change it indeed.

Best regards,
Pierre
